src/main/app/controller/GetController.ts

```diff
@@ -12,15 +12,22 @@ export class GetController {
   ) {}
 
   public async get(req: AppRequest, res: Response): Promise<void> {
-    const languageContent = this.content[req.session.lang] || this.content['en'] || {};
-    const commonLanguageContent = commonContent[req.session.lang] || commonContent['en'];
-    const commonPageContent = this.content.common || {};
+    setImmediate(async () => {
+      if (res.headersSent) {
+        // If there's an async error, it may have already rendered an error page upstream,
+        // so we don't want to call render again if headers have already been sent
+        return;
+      }
```

Comment:
I'm sorry, I don't think this is a viable solution. I'm not willing to wrap all controllers in a setImmediate. We can switch to express 5.x alpha if we have to, but there's got to be a better way

Reply:
👍 I've removed `setImmediate` in https://github.com/hmcts/nfdiv-frontend/pull/107/commits/719bddc3bb808d05519972616381ea2609e9fc98